pkg/monitors/processlist/processlist_windows.go

```diff
@@ -137,11 +158,40 @@ func ProcessList(conf *Config, cache *osCache) ([]*TopProcess, error) {
 			VirtualMemoryBytes:  p.VirtualSize,
 			WorkingSetSizeBytes: p.WorkingSetSize,
 			SharedMemBytes:      0,
-			Status:              *p.Status,
+			Status:              status,
 			MemPercent:          memPercent,
 			TotalCPUTime:        totalTime,
 			Command:             command,
 		})
 	}
 	return procs, nil
 }
+
+// Mapping each thread's state to its respective process.
+// for example, threadList = []Win32Thread{{ProcessHandle: "1", ThreadState: 3},
+// {ProcessHandle: "2", ThreadState: 3},{ProcessHandle: "1", ThreadState: 5},{ProcessHandle: "1", ThreadState: 5},}
+// it returns map[string][]uint32{"1": []uint32{3, 5, 5}, "2": []uint32{3},},
+func mapThreadToProcess(threadList []Win32Thread) map[string][]uint32 {
+	var processes = make(map[string][]uint32)
+	for _, thread := range threadList {
+		processes[thread.ProcessHandle] = append(processes[thread.ProcessHandle], thread.ThreadState)
+	}
+	return processes
+}
+
+// Returns the process status depending upon all thread's state.
+// if all the threads of a process are in waiting state then it returns "S"(sleeping)
+// else it returns "R"(running)
+func statusMapping(threadStates []uint32) string {
+	if len(threadStates) == 0 {
+		return ""
+	}
+
+	for _, state := range threadStates {
+		// threadState = 5 represents the thread is in waiting state
+		if state != 5 {
```

Comment:
I think this would a more readable if this `5` was a const variable

Reply:
Pushed the changes accordingly.